packages/sdk-js/src/GrowthBook.ts

```diff
@@ -683,6 +698,7 @@ export class GrowthBook<
     if (isBlocked) {
       process.env.NODE_ENV !== "production" &&
         this.log("Auto experiment blocked", { id: experiment.key });
+      this._unsetAntiFlicker();
```

Comment:
This doesn't feel like it belongs in this method.  Imagine you have 2 redirect experiments.  The first one is blocked, so it hits this and removes the overlay.  The next one is not blocked so it redirects, but the overlay is already gone

Reply:
Sure we can remove. The flip side though is the anti-flicker stays up far too long (3000ms by default) when only a single experiment triggers but is blocked.

Reply:
there are a few other bail-out scenarios where i'm calling `_unsetAntiFlicker();`:
- redirect variant does not actually redirect (ex: control)
- invalid urlPatterns (rare)
- already on the target page (rare)

the same double experiment scenario you mentioned earlier could happen here too, right?